src/bin/build-keycloak-theme/argv.ts

```diff
@@ -0,0 +1,9 @@
+export function argv(key: string) {
+    if (process.argv.includes(`--${key}`)) return true;
+
+    const value = process.argv.find(element => element.startsWith(`--${key}=`));
+
+    if (!value) return "";
```

Comment:
sorry randomly got here why is this not returning false in contrast to line 2,

Reply:
@satanshiro,  
It should indeed?  
Are you reviewing this PR?  
It's not mine and have be stale for almost a year.  
If you submit another PR I'll review, merge and release promptly.  

Best

Reply:
oh wow I didn't notice the dates I was just wondering what work is being done, while I was checking to see if otp pages are available on keycloakify and it is. don't think I'll be needing this pr. thanks!